buildSrc/src/test/java/com/osacky/flank/gradle/YamlWriterTest.kt

```diff
@@ -790,6 +807,22 @@ class YamlWriterTest {
     assertTrue(properties.contains("  num-uniform-shards: 20"))
   }
 
+  @Test
+  fun writeOutputStyle() {
+    val properties = emptyExtension {
+      outputStyle.set("anyString")
```

Comment:
i guess this will fail when the flank command is executed?
i'm fine with that, just wanted to point it out. i don't think validating this option on the fladle side makes sense in case more options are introduced by flank down the line.

Reply:
Yes, I did that on purpose for the same reason you pointed out.